src/transactions/utils/index.js

```diff
@@ -25,3 +25,4 @@ export {
 	verifyTransaction,
 } from './signAndVerify';
 export { getTimeFromBlockchainEpoch, getTimeWithOffset } from './time';
+export { validatePublicKey, validatePublicKeys } from './validatePublicKey';
```

Comment:
We can't do `export * from './validatePublicKey';` until we support Babel 7 because of babel-transform, correct?

Reply:
Yes, looks like it
https://github.com/babel/babel/issues/2877